bundles/org.openhab.binding.shelly/README.md

```diff
@@ -517,6 +534,28 @@ The Thing id is derived from the service name, so that's the reason why the Thin
 |          |totalKWH     |Number   |yes      |Total energy consumption in Watts since the device powered up (resets on restart)|
 |          |lastUpdate   |DateTime |yes      |Timestamp of the last measurement                                                |
 
+### Shelly 2 - roller mode thing-type: shelly2-roller)
+
+|Group     |Channel      |Type     |read-only|Description                                                                           |
+|----------|-------------|---------|---------|--------------------------------------------------------------------------------------|
+|roller    |control      |Rollershutter|r/w  |can be open (0%), stop, or close (100%); could also handle ON (open) and OFF (close)  |
+|          |input        |Switch   |yes      |ON: Input/Button is powered, see General Notes on Channels                            |
+|          |event        |Trigger  |yes      |Roller event/trigger with payload ROLLER_OPEN / ROLLER_CLOSE / ROLLER_STOP            |
+|          |rollerpos    |Number   |r/w      |Roller position: 100%=open...0%=closed; gets updated when the roller stops, see Notes |
```

Comment:
Is there any reason introducing a second channel representing the position in percent? Rollershutter can also be fed and controlled with PercentType.

Reply:
this has a long story, some of the UI integration require 0..100% vs. 100..0%, The rollershutter's one is default and the rollerpos is marked as an advanced channel.